### PR TITLE
Add support to disable SSL validation over HTTPS

### PR DESCRIPTION
**Pull request by mikethebeer**
This enables the possiblity to disable SSL validation in the host URI.

e.g. `--hosts https://localhost:4200/?verify_ssl=false`

This PR is a different approach to https://github.com/mfussenegger/cr8/pull/242

### CODE REVIEW

File: cr8/clients.py [hunk 1]
```diff
@@ -239,7 +268,8 @@ class HttpClient:
     def __init__(self, hosts, conn_pool_limit=25):
         self.hosts = hosts
         self.urls = itertools.cycle([i + '/_sql' for i in hosts])
-        conn = aiohttp.TCPConnector(limit=conn_pool_limit)
+        self.verify_ssl = _verify_ssl(self.hosts)
```
Comment (mfussenegger) [inline cr8/clients.py]: Could you remove the `self.` assignment and keep it a local variable?

File: cr8/clients.py [hunk 2]
```diff
@@ -172,6 +173,34 @@ def _to_dsn(hosts):
...
+def _to_boolean(v):
+    if str(v).lower() in ("yes", "true", "t", "1"):
+        return True
+    elif str(v).lower() in ("no", "false", "f", "0"):
+        return False
+    else:
+        raise ValueError('not a boolean value')
+
+
+def _verify_ssl(hosts):
```
Comment (mfussenegger) [inline cr8/clients.py]: Maybe rename to `_verify_ssl_from_first` to clarify that it takes the value of the first explicitly specified `verify_ssl` flag?

File: cr8/clients.py [hunk 3]
```diff
@@ -172,6 +173,34 @@ def _to_dsn(hosts):
     return dsn
 
 
+def _to_boolean(v):
+    if str(v).lower() in ("yes", "true", "t", "1"):
```
Comment (mfussenegger) [inline cr8/clients.py]: I'd make this stricter and just support `true` and `false`, or did you do it that way for compatibility with something else?

**Comment by mikethebeer** [comment]
@mfussenegger addressed your comments

**Comment by mfussenegger** [comment]
Thanks, merged it.

**Pull request merged**